### PR TITLE
[New feature] URL card should have copy button

### PR DESCRIPTION
**Pull request by svnnynior**
According to #8,
What I did
- add a copy button
- remove https:// prefix in the url card
- add notification popup when successfully copied

### CODE REVIEW

File: client/src/components/Home/Home.ts [hunk 1]
```diff
@@ -65,10 +65,10 @@ export default Vue.extend({
         }
     },
     mounted() {
-        axios
-            .get(API_URL)
-            .then(response => this.urls = response.data)
-            .catch(error => this.errorMessage = 'hmm... It seems network is broken or server is down 😴');
```
Comment (sprakash57) [inline client/src/components/Home/Home.ts:71]: First of all, thank you for your efforts. I can see you have removed a few lines. Any specific reason for that? I am curious how the application will behave if there isn't any network?

Comment (svnnynior) [reply client/src/components/Home/Home.ts]: oh right, my bad, I forgot to put back the comment. fixed now.

Comment (sprakash57) [reply client/src/components/Home/Home.ts]: Awesome.

**Comment by sprakash57** [comment]
PR merged. If you enjoyed your time here, Please star the repo.

**Pull request merged**